app/models/fika.rb

```diff
@@ -2,4 +2,15 @@ class Fika < ApplicationRecord
   validates_presence_of :start_date, :end_date
 
   has_and_belongs_to_many :participants, join_table: 'fikas_participants'
+
+  def self.participants_uniq_matcher
+    participants = Participant.all
+    arr = participants.to_a.permutation(2).to_a
+    arr_sorted = []
+    arr.each do |pair|
+      arr_sorted.push(pair.sort_by(&:id))
+    end
+    arr_sorted.uniq!
+    arr_sorted
+  end
```

Comment:
Can't say I understand everything here but it looks awesome!

Reply:
`.uniq!` actually saves the value of `arr_sorted` to only store unique values (in this case two item arrays of ActiveRecord instances). Line 14 could be clarified with a `return` statement but as you know it would be removed by rubocop as it's not needed in Ruby.